build/charts/antrea/README.md

```diff
@@ -97,6 +97,11 @@ Kubernetes: `>= 1.16.0-0`
 | nodePortLocal.portRange | string | `"61000-62000"` | Port range used by NodePortLocal when creating Pod port mappings. |
 | ovs.bridgeName | string | `"br-int"` | Name of the OVS bridge antrea-agent will create and use. |
 | ovs.hwOffload | bool | `false` | Enable hardware offload for the OVS bridge (required additional configuration). |
+| secondaryNetworkOVS.ovsDatapathType | string | `"system"` | 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run OVS in userspace mode. Userspace mode requires the tun device driver to be available. |
+| secondaryNetworkOVS.ovsIntBrName | string | `"br-secnet-int"` | Name of the secondary network OVS bridge1 antrea-agent's podWatch controller will create and use. |
+| secondaryNetworkOVS.ovsPatchPort | string | `"br-secnet-patch0"` | Name of the OVS patch port which would connected two OVS bridges per secondary interface config. |
+| secondaryNetworkOVS.ovsTransBrName | string | `"br-secnet-trans"` | Name of the secondary network OVS bridge2 antrea-agent's podWatch controller will create and use. |
```

Comment:
Can we update all the descriptions to be consistent with antrea-agent-secondary-network.conf?